client/css/style.css

```diff
@@ -1492,18 +1504,16 @@ kbd {
 
 .context-menu-divider {
 	height: 1px;
-	margin: 6px 0;
 	background-color: rgba(0, 0, 0, .1);
 }
 
 .context-menu-item,
 .textcomplete-item {
 	cursor: pointer;
 	display: block;
-	padding: 4px 8px;
+	padding: 10px 8px;
 	color: #333;
-	margin-top: 6px;
-	margin-bottom: 6px;
```

Comment:
Correct, the separator (`divider` in our HTML) is not meant to be present between all rows. The nearest and easiest addition to these dropdowns I can think of is having multiple user actions: kick, ban, etc. So rows would be: nick / divider / ban / kick / ...